string/strfry.c

```diff
@@ -35,13 +35,12 @@ strfry (char *string)
       init = 1;
     }
 
-  size_t len = strlen (string);
```

Comment:
It has to iterate over the whole string anyways, getting the length of it is a waste of time, can just iterate to the null char instead (strfry does that anyways, so you're effectively iterating over the string twice).